apps/trading/e2e/README.md

```diff
@@ -0,0 +1,118 @@
+# Trading End-To-End Tests
+
+This direcotry contains end-to-end tests for the trading application using vega-market-sim. This README will guide you through setting up your environment and running the tests.
+
+## Prerequisites
+
+- [Poetry](https://python-poetry.org/docs/#installing-with-the-official-installer)
+- [Docker](https://www.docker.com/)
+- [Python versions ">=3.9,<3.11"](https://www.python.org/)
+
+## Getting Started
+
+1. **Install Poetry**: Follow the instructions on the [official Poetry website](https://python-poetry.org/docs/#installing-with-the-official-installer).
+2. **Install Docker**: Follow the instructions on the [official Docker website](https://docs.docker.com/desktop/).
+3. **Install Python**: Follow the instructions on the [official Python website](https://www.python.org/)
+   **ensure you install a version between 3.9 and 3.11.**
+4. **Start up a Poetry environment**: Execute the commands below to configure the Poetry environment.
+
+```bash
+poetry shell
+```
+
+5. **Install python dependencies**
+
+```bash
+poetry install
+```
+
+6. **Install Playwright Browsers**: Execute the command below to browsers for Playwright.
+
+```bash
+playwright install chromium
+```
+
+7. **Download necessary binaries**:
+   Use the following command within your Python environment. The `--force` flag ensures the binaries are overwritten, and the `--version` specifies the desired version. e.g. `v0.73.1`
+
+```bash
+python -m vega_sim.tools.load_binaries --force --version $VEGA_VERSION
+```
+
+8. **Pull the desired Docker image**
+
+```bash
+docker pull vegaprotocol/trading:develop
+```
+
+9. **Run tests**: Poetry/Python will serve the app from docker
+
+```bash
+poetry run pytest
```

Comment:
to run this command successfully on wsl ubuntu I had to explicitly pull vegaprotocol/trading:latest, otherwise it errored when attempting to create the container.